main.c

```diff
@@ -570,7 +600,7 @@ ImportConfigFile()
             {
                 if (GetLastError() == ERROR_FILE_EXISTS)
                 {
```

Comment:
As the main purpose of this is for ease of distributing configs, don't we want to allow optional overwrite of an existing config?